R/remote-info.R

```diff
@@ -0,0 +1,32 @@
+getRemoteInfo <- function(pkgRecord) {
+  if (pkgRecord$source == "github") {
+    remote_info <- as.data.frame(
```

Comment:
The assigned variable is unnecessary; you can directly return the data frame.

Reply:
I made this change.